### PR TITLE
Add tests for the content_types example

### PR DESCRIPTION
**Pull request by sethlopezme**
Relates to #62.

### CODE REVIEW

File: examples/content_types/src/tests.rs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+use super::rocket;
+use super::serde_json;
+use super::Person;
+use rocket::http::{ContentType, Method, Status};
+use rocket::testing::MockRequest;
+
+fn test(uri: &str,
```
Comment (SergioBenitez) [inline examples/content_types/src/tests.rs]: Can we shorten the variable names so the declaration fits in one line? `status` and `body` are fine.

File: examples/content_types/src/tests.rs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+fn test_hello() {
+    let person = Person {
+        name: "Michael".to_string(),
+        age: 80,
+    };
+
+    test("/hello/Michael/80",
+         ContentType::JSON,
+         Status::Ok,
+         serde_json::to_string(&person).unwrap());
```
Comment (SergioBenitez) [inline examples/content_types/src/tests.rs]: Can we move this into its own line (as a declaration) to fit the `test` call into one line?

File: examples/content_types/src/tests.rs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+         format!("<p>This server only supports JSON requests, not '{}'.</p>",
+                 ContentType::HTML));
+}
+
+#[test]
+fn test_404() {
+    test("/unknown",
+         ContentType::JSON,
+         Status::NotFound,
+         "<p>Sorry, '/unknown' is not a valid path!</p>
```
Comment (SergioBenitez) [inline examples/content_types/src/tests.rs]: This is pretty mysterious spacing, especially with the rogue '/' in there. Consider changing the actual route somehow to make this clearer.

Comment (sethlopezme) [inline examples/content_types/src/tests.rs]: Ha, yeah, rustfmt was adamant about formatting it this way. I'll change it up so it's not so crazy.

**Comment by SergioBenitez** [comment]
Waiting on these changes. :)

**Comment by sethlopezme** [comment]
Sorry, I'll try to get to this later today!

**Comment by sethlopezme** [comment]
I've implemented the changes you requested!

**Comment by SergioBenitez** [comment]
Merged in dec585dbd4e3a03ab0189d2c1aac669f867b5ea9.

**Pull request closed without merging**